addons/sale/__manifest__.py

```diff
@@ -29,6 +29,7 @@
         'data/mail_template_data.xml',
         'data/sale_tour.xml',
         'data/ir_config_parameter.xml', # Needs mail_template_data
+        'data/portal_card_data.xml',
```

Comment:
Please put it after `'data/mail_template_data.xml',`, data are alphabetically ordered (except some files requiring other to be loaded first).